Updated runtime to 42

spanderman opened the pull request:
Runtime 41 is now end-of-life according to flatpak:

Info: org.gnome.Platform//41 is end-of-life, with reason:
   The GNOME 41 runtime is no longer supported as of September 17, 2022. Please ask your application developer to migrate to a supported platform.
Applications using this runtime:
   org.gnome.Cheese

Not sure why this source was already version 42, instead:

https://download.gnome.org/sources/gnome-desktop/42/gnome-desktop-42.4.tar.xz

spanderman committed:
Updated runtime to 42

Runtime 41 is now end-of-life according to flatpak:

Info: org.gnome.Platform//41 is end-of-life, with reason:
   The GNOME 41 runtime is no longer supported as of September 17, 2022. Please ask your application developer to migrate to a supported platform.
Applications using this runtime:
   org.gnome.Cheese

bbhtt commented:
You need to add clutter from shared-modules to the manifest, it was removed from 42.

spanderman committed:
added clutter shared module

As per:

https://github.com/flathub/org.gnome.Cheese/pull/23#issuecomment-1286342455

Now in shared modules:

https://github.com/flathub/shared-modules/blob/master/clutter/clutter.json

spanderman commented:
build

Commit:
updated shared-modules git submodule to latest from master

spanderman commented:
build

spanderman commented:
> You need to add clutter from shared-modules to the manifest, it was removed from 42.

@bbhtt my bad, didn't notice that

Added the shared module, fixed the pinned git submodule commit and now build passed.

spanderman commented:
@bbhtt tested locally, didn't notice any issue so far, compared to the latest stable version.

Steinhagen commented:
Can confirm, works as expected after the runtime update.

marcthe12 commented:
Any reason version 42 not 43 since that has been released?

spanderman commented:
@marcthe12 was just the minimum effort choice. Not sure I am able to change much more, if required.

Given that this is not merged yet, I can give it a try, I guess.

spanderman commented:
@marcthe12  plus Cheese itself is still on release 41, fyi (there's a 43, still an alpha), and again I am not fully sure if there's any implication I miss:

https://download.gnome.org/sources/cheese/

spanderman commented:
Created another pull request, trying with GNOME runtime 43, being the latest available.

https://github.com/flathub/org.gnome.Cheese/pull/24

@marcthe12 not sure it's going to work, though. Seems to me there are changes to do in Cheese itself, before we can proceed with the runtime 43 adoption. Makes sense?

nedrichards commented:
bot, build

spanderman commented:
Thanks @nedrichards
